backend/reservations/views/admin.py

```diff
@@ -142,144 +144,112 @@ def guest_search(arr, low, high, x):
+        time.sleep(5)
+        if os.environ.get('ROOMBAHT_SEND_MAIL', 'FALSE').lower() == 'true':
+            apppass = os.environ['ROOMBAHT_EMAIL_HOST_PASSWORD']
+            logger.debug(f'[+] Sending invite for guest {guest_new["first_name"]} {guest_new["last_name"]}')
+
+            body_text = f"""
+                BleepBloopBleep, this is the Room Service RoomBaht for Room Swaps letting you know the floors have been cleaned and you have been assigned a room. No bucket or mop needed.
+                
+                After you login below you can view your current room, look at other rooms and send trade requests. This functionality is only available until Monday 11/7 at 5pm PST, so please make sure you are good with what you have or trade early.
+                
+                Goes without saying, but don't forward this email.
+                
+                This is your password, there are many like it but this one is yours. Once you use this password on a device, RoomBaht will remember you, but only on that device.
+                Copy and paste this password. Because let’s face it, no one should trust humans to make passwords:
+                {otp}
+                http://rooms.take3presents.com/login
+                
+                Good Luck, Starfighter.
+
+            """
+
+            send_mail("RoomService RoomBaht",
+                      body_text,
+                      "placement@take3presents.com",
+                      [guest_new["email"]],
+                      auth_user="placement@take3presents.com",
+                      auth_password=apppass,
+                      fail_silently=False,)
 
 
 def create_guest_entries(init_file=""):
-    dr = None
-    new_guests=[]
-    gl = list(Guest.objects.all())
-    guest_sort(gl)
+    guest_rows = []
+    guests_existing = list(Guest.objects.all())
+    guest_sort(guests_existing)
     try:
-        glen = len(gl)-1
+        glen = len(guests_existing)-1
     except TypeError as e:
         logger.warning(f"No existing guests to show ")
         glen=0
 
     with open(init_file, "r") as f1:
-        dr = []
-        for elem in DictReader(f1):
-            dr.append(elem)
+        for row in DictReader(f1):
+            stripd = {k.lstrip().rstrip(): v.lstrip().rstrip() for k, v in row.items() if type(k)==str and type(v)==str}
+            guest_rows.append(stripd)
 
     characters = string.ascii_letters + string.digits + string.punctuation
-
-    for guest_new in dr:
-        logger.debug(f'new guest: {guest_new}')
-        guest_entries = Guest.objects.filter(email=guest_new["email"])
-        if ("Directed Sale Room (1 King Bed)" in guest_new["product"]):
-            guest_new["product"] = "King"
-        if ("Directed Sale Room (1 Queen Bed)" in guest_new["product"]):
-            guest_new["product"] = "Queen"
-
-        if ("Hard Rock" in guest_new["product"]):
+    for guest_new in guest_rows:
+        if(guest_new['product'][:3] == "Art"):
```

Comment:
What's the context behind ignoring art users? And the hard rock users? Shouldn't we still put them in the DB (for hotel export) but filter them out as needed?

Reply:
Yea leaving out for now bc is just more data and log noise. Would be nice to have, "All the users and all the data" 
Maybe after we know all the updates and reports are worky